kanidmd/src/lib/actors/v1_read.rs

```diff
@@ -394,7 +398,7 @@ impl QueryServerReadV1 {
                             // From the entry, turn it into the value
                             .and_then(|e| {
                                 e.get_ava_single("radius_secret")
-                                    .and_then(|v| v.get_radius_secret().map(|s| s.to_string()))
+                                    .and_then(|v| v.get_secret_str().map(|s| s.to_string()))
```

Comment:
You should consider adding this as a possible lint for clippy?

Reply:
> You should consider adding this as a possible lint for clippy?

How do I do that? I found the lint that corresponds to this here:

https://rust-lang.github.io/rust-clippy/master/#redundant_closure_for_method_calls

Reply:
I think we already apply this check - what I'm saying is taht because this isn't a redundant closure (because it's a struct method), you should submit a literal bug to clippy itself to request this specific case to be added.